### PR TITLE
Move CSS system color keywords to their own page

### PR DESCRIPTION
**Pull request by teoli2003**
Part of https://github.com/openwebdocs/project/issues/92 (revamp our color guide and reference).

This PR moves the system-related color keywords into their own page (taking them off the `<color>` page).

Note:
- There is much more work to be done on the `<color>` page, I intentionally limited this PR by moving the color keywords only.
- We don't have BCD info for these, so I didn't add a _Browser compatibility_ section to it (I'll see if I have time to come back to this during this project, but this is secondary).

### CODE REVIEW

File: files/en-us/web/css/color_value/system_color_keywords/index.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
+---
+title: System color keywords
+slug: Web/CSS/color_value/System_color_keywords
```
Comment (dipikabh) [inline files/en-us/web/css/color_value/system_color_keywords/index.md:3]:
```suggestion
slug: Web/CSS/color_value/system_color_keywords
```
To match casing in slug here: "https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color_keywords"

File: files/en-us/web/css/color_value/system_color_keywords/index.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+- `ThreeDShadow` {{deprecated_inline}}
+  - : The color of the lighter (generally inner) of the two borders away from the light source for 3-D elements that appear 3-D due to two concentric layers of surrounding border.
+- `Window` {{deprecated_inline}}
+  - : Window background. Should be used with the `WindowText` foreground color.
+- `WindowFrame` {{deprecated_inline}}
+  - : Window frame.
+- `WindowText` {{deprecated_inline}}
+  - : Text in windows. Should be used with the `Window` background color.
+
+## Firefox non-standard System Color Extensions
```
Comment (dipikabh) [inline files/en-us/web/css/color_value/system_color_keywords/index.md:100]:
```suggestion
## Firefox non-standard system color extensions
```

File: files/en-us/web/css/color_value/system_color_keywords/index.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+Firefox also support some non-standard extensions to the system color keyword set.
```
Comment (dipikabh) [inline files/en-us/web/css/color_value/system_color_keywords/index.md:102]:
```suggestion
Firefox also supports some non-standard extensions to the system color keyword set. 
```

File: files/en-us/web/css/color_value/system_color_keywords/index.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+  - : Accent colors.
+- `-moz-mac-chrome-active` {{non-standard_inline}}, `-moz-mac-chrome-inactive` {{non-standard_inline}}
+  - : Colors for inactive and active browser chrome.
+- `-moz-mac-focusring` {{non-standard_inline}}, `-moz-mac-menuselect` {{non-standard_inline}}, `-moz-mac-menushadow` {{non-standard_inline}}, `-moz-mac-menutextselect` {{non-standard_inline}}, `-moz-MenuHover` {{non-standard_inline}}
+  - : Background color for hovered menu items. Often similar to `Highlight`. Should be used with the `-moz-MenuHoverText` or `-moz-MenuBarHoverText` foreground color.
+- `-moz-MenuHoverText` {{non-standard_inline}}
+  - : Text color for hovered menu items. Often similar to `HighlightText`. Should be used with the `-moz-MenuHover` background color.
+- `-moz-MenuBarText` {{non-standard_inline}}
+  - : Text color in menu bars. Often similar to `MenuText`. Should be used on top of `Menu` background.
+- `-moz-MenuBarHoverText
```
Comment (dipikabh) [inline files/en-us/web/css/color_value/system_color_keywords/index.md:138]:
```suggestion
- `-moz-MenuBarHoverText`  {{non-standard_inline}}
```

File: files/en-us/web/css/color_value/system_color_keywords/index.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+- `-moz-win-mediatext` {{non-standard_inline}}
+  - : Should be used for text in objects with `{{cssxref("appearance")}}: -moz-win-media-toolbox`.
+- `-moz-win-accentcolor` {{non-standard_inline}}
+  - : Used to access the Windows 10 custom accent color that you can set on the start menu, taskbar, title bars, etc.
+- `-moz-win-accentcolortext` {{non-standard_inline}}
+  - : Used to access the color of text placed over the Windows 10 custom accent color in the start menu, taskbar, title bars, etc.
+
+## See also
+
+- [`<color>`](/en-US/docs/Web/CSS/color_value) the data type these keywords belong to.
```
Comment (dipikabh) [inline files/en-us/web/css/color_value/system_color_keywords/index.md:155]:
```suggestion
- [`<color>`](/en-US/docs/Web/CSS/color_value) data type these keywords belong to.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]

**Review by dipikabh — APPROVED** [review]
Looks good! 👍 Thank you.

**Comment by alialh600pp** [comment]
`files/en-us/mozilla/add-ons/webextensions/api/scripting/injectionresult/index.md`

**Pull request merged**